### DISCUSSION TITLE
Discord channel

### DISCUSSION THREAD

**Discussion by caniko** [pydantic/pydantic#3533] [Feature Request]
Community building, rapid discussions, memes, and a place to share your love for Pydantic and the rest of Python.

**Comment by daveraja** [comment]
Does the Pydantic community still use the Discord server?

The invitation URL https://discord.com/invite/FXTHeFtgtg   linked from https://docs.pydantic.dev/latest/help_with_pydantic/ seems to be invalid or expired.

**Comment by unclemusclez** [comment]
+1
